.github/pull_request_template.md

```diff
@@ -0,0 +1,28 @@
+## Reason for This PR
+
+`[Author TODO: add issue # or explain reasoning.]`
+
+## Description of Changes
+
+`[Author TODO: add description of changes.]`
+
+## License Acceptance
+
+By submitting this pull request, I confirm that my contribution is made under
+the terms of the Apache 2.0 license.
+
+## PR Checklist
+
+`[Author TODO: Meet these criteria. Where there are two options, keep one.]`  
+`[Reviewer TODO: Verify that these criteria are met. Request changes if not]`
+
+- [ ] All commits in this PR are signed (`git commit -s`).
+- [ ] The PR is either linked to an issue, or a reason is provided. 
+- [ ] The description of changes is clear and encompassing.
+- [ ] Either no docs need to be updated as part of this PR, or, the required
+      changes are included in this PR. Docs in scope are all `*.md` files
+      located either in the repository root, or in the `docs/` directory.
```

Comment:
We also have documentation in the root of each crate for some crates. One example is micro_http where recently we had a PR that didn't update that documentation. Should we also mention this type documentation (in-crate documentation)?